catalogue_api/api/src/test/scala/uk/ac/wellcome/platform/api/works/v2/ApiV2WorksIncludesTest.scala

```diff
@@ -110,16 +99,300 @@ class ApiV2WorksIncludesTest extends ApiV2WorksTestBase {
                               | "type": "Work",
                               | "id": "${work.canonicalId}",
                               | "title": "${work.title}",
-                              | "contributors": [ ],
-                              | "items": [ ${items(work.items)} ],
-                              | "subjects": [ ],
-                              | "genres": [ ],
-                              | "production": [ ]
+                              | "items": [ ${items(work.items)} ]
                               |}
           """.stripMargin
           )
         }
     }
   }
 
+  it(
+    "includes a list of subjects on a list endpoint if we pass ?include=subjects") {
+    withV2Api {
+      case (apiPrefix, _, indexNameV2, itemType, server: EmbeddedHttpServer) =>
+        val works = createIdentifiedWorks(count = 2).sortBy { _.canonicalId }
+
+
+        val subjects1 = List(Subject("ornitology", List(Unidentifiable(Concept("ornitology")))))
+        val subjects2 = List(Subject("flying cars", List(Unidentifiable(Concept("flying cars")))))
+        val work0 = works(0).copy(subjects = subjects1)
+        val work1 = works(1).copy(subjects = subjects2)
+
+        insertIntoElasticsearch(indexNameV2, itemType, work0, work1)
+
+        eventually {
+          server.httpGet(
+            path = s"/$apiPrefix/works?include=subjects",
+            andExpect = Status.Ok,
+            withJsonBody =
+              s"""
+                 |{
+                 |  ${resultList(apiPrefix, totalResults = 2)},
+                 |  "results": [
+                 |   {
+                 |     "type": "Work",
+                 |     "id": "${work0.canonicalId}",
+                 |     "title": "${work0.title}",
+                 |     "subjects": [ ${subjects(subjects1)}]
+                 |   },
+                 |   {
+                 |     "type": "Work",
+                 |     "id": "${work1.canonicalId}",
+                 |     "title": "${work1.title}",
+                 |     "subjects": [ ${subjects(subjects2)}]
+                 |   }
+                 |  ]
+                 |}
+          """.stripMargin
+          )
+        }
+    }
+  }
+
+  it(
+    "includes a list of subjects on a single work endpoint if we pass ?include=subjects") {
+    withV2Api {
+      case (apiPrefix, _, indexNameV2, itemType, server: EmbeddedHttpServer) =>
+
+        val subject = List(Subject("ornitology", List(Unidentifiable(Concept("ornitology")))))
+        val work = createIdentifiedWork.copy(subjects = subject)
+
+        insertIntoElasticsearch(indexNameV2, itemType, work)
+
+        eventually {
+          server.httpGet(
+            path = s"/$apiPrefix/works/${work.canonicalId}?include=subjects",
+            andExpect = Status.Ok,
+            withJsonBody =
+              s"""
+                 |{
+                 |"@context": "https://localhost:8888/$apiPrefix/context.json",
```

Comment:
minor: could you clean up the indentation on this JSON please? Makes it easier to read.